### ISSUE TITLE
Allow for configurable / plugable cache backends

### ISSUE THREAD

**Issue by cdeszaq** [grygoriy/bruteforcedefender#2]
It would be great to be able to easily plug in alternative cache backends, like Redis, so that distributed environments (ie. load-balanced across multiple machines) can use this plugin.

**Comment by grygoriy** [comment]
yes, this is plan for next version. Unfortunately without any release date for now
